tag.go

```diff
@@ -51,7 +51,15 @@ func SortTags(tags []Tag) []Tag {
 	return deduplicateTags(tags)
 }
 
-func tagIsLess(a, b Tag) bool { return a.Name < b.Name }
+func tagIsLess(a, b Tag) int {
```

Comment:
I recommend you call this `compareTags`, since `less` implies a returned bool, which is no longer the case.